src/RapidezServiceProvider.php

```diff
@@ -243,6 +245,10 @@ protected function bootMacros(): self
             })->sortByDesc('store_id')->first();
         });
 
+        ViewView::macro('renderOneliner', function () {
+            return trim(preg_replace('/\s+/', ' ', $this->render()));
+        });
+
```

Comment:
I think this might not be necessary if we use https://laravel.com/docs/10.x/strings#method-str-squish.
`Str::squish(view('rapidez::cart.queries.cart')->render())`
Depends on if we want to call it on the view or if using it more verbose is okay